js/hotgraphicPopupView.js

```diff
@@ -0,0 +1,189 @@
+define([
+    'core/js/adapt'
+], function(Adapt) {
+    'use strict';
+
+    var HotgraphicPopupView = Backbone.View.extend({
+
+        className: 'hotgraphic-popup',
+
+        events: {
+            'click .hotgraphic-popup-done': 'closePopup',
+            'click .hotgraphic-popup-nav .back': 'onBackClick',
+            'click .hotgraphic-popup-nav .next': 'onNextClick'
+        },
+
+        initialize: function() {
+            _.bindAll(this, 'onKeyUp');
+            this.listenToOnce(Adapt, "notify:opened", this.onOpened);
+            this.render();
+        },
+
+        onOpened: function(notifyView) {
+            this.setupEscapeKey();
+            this.trigger('popup:open');
+
+            var currentIndex = this.model.getActiveItem().get('_index');
+
+            this.$('.hotgraphic-popup-inner').a11y_on(false);
+            this.$('.hotgraphic-item').hide().removeClass('active');
```

Comment:
none of the `.hotgraphic-item` divs have an `.active` class at this point so `removeClass('active')` would appear to be redundant, as would the next line (31)

Reply:
similar code exists in both `previousHotGraphic` and `nextHotGraphic` so it might be a good idea to refactor this out into a separate method that takes `currentIndex` as a parameter